sdk/core/src/services/ApiService.ts

```diff
@@ -130,7 +130,7 @@ export default class ApiService {
     let jsonResponse
 
     if (status.toString().startsWith('2')) {
-      jsonResponse = await response.json()
+      jsonResponse = status.toString() === '204' ? {} : await response.json()
```

Comment:
It does not seem to affect anything because we don't actually use its return value anywhere in this PR?

Reply:
not really is you try to get json from empty response you fail with an error